src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,32 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() throws ServletException {
+        myResource = MyCoolResource.openResource();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String localDateTime = LocalDateTime.now().toString();
+        myResource.write(localDateTime);
+        req.setAttribute("date", localDateTime);
+        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+    }
+
+    @Override
+    public void destroy() {
+        myResource.close();
+    }
```

Comment:
```suggestion
        myResource.close();
        super.destroy();
    }
```
"Do not forget to call init() and destroy() methods from the superclass when we override them"

Reply:
Mentor said, that it's not necessarily